policy/modules/apps/gpg.te

```diff
@@ -83,6 +83,7 @@ allow gpg_t self:process { signal signull setrlimit getcap setcap getsched setsc
 dontaudit gpg_t self:netlink_audit_socket r_netlink_socket_perms;
 allow gpg_t self:fifo_file rw_fifo_file_perms;
 allow gpg_t self:tcp_socket { accept listen };
+allow gpg_t self:unix_stream_socket { accept listen };
```

Comment:
Please use `create_stream_socket_perms`.

Reply:
I can't foresee a valid reason for doing so, please see https://github.com/SELinuxProject/refpolicy/pull/639#discussion_r1316142515

Reply:
I have updated the discussion, please see https://github.com/SELinuxProject/refpolicy/pull/639#discussion_r1316346698

Reply:
Fixed after source code audit.